### ISSUE TITLE
Syncfusion ListViewControl problem

### ISSUE THREAD

**Issue by powerdude** [adospace/mauireactor-integration#3]
Hi,

Not sure if this is intentional or not or if I'm missing something, but it seems when scaffolding the `SfListView` control, there are no wrappers for things the the `LoadMoreTemplate` or the `LoadMoreCommand`.  What is the best way to configure these properties?

**Comment by powerdude** [comment]
Also `ItemsSource` is missing.

**Comment by adospace** [comment]
Hi,
I've upgraded the Synfusion sample app including the SfListView: I've added the implementation for the GroupHeaderTemplate, LoadMoreTemplate, and LoadMore command.
You can use it as a model to add the templates/commands you may need.

**Comment by powerdude** [comment]
Hi, just following up on this.  I tried your code but am getting an error for the `OnMigrated` method since it already exists in the scaffold generated code.

```csharp
   protected override void OnMigrated(VisualNode newNode)
        {
            var newSfListView = ((SfListView<T>)newNode);
            newSfListView._customDataTemplate = _customDataTemplate;
            if (newSfListView._customDataTemplate != null)
            {
                newSfListView._customDataTemplate.Owner = newSfListView;
            }

            base.OnMigrated(newNode);
        }
```

**Comment by adospace** [comment]
Are you using MauiReactor version 2 or 3?

**Comment by powerdude** [comment]
2

**Comment by adospace** [comment]
Fixed in MauiReactor 2.0.58
